charge.go

```diff
@@ -512,6 +512,11 @@ type ChargePaymentMethodDetailsP24 struct {
 	Reference    string `json:"reference"`
 	VerifiedName string `json:"verified_name"`
 }
+type ChargePaymentMethodDetailsSepaCreditTransfer struct {
+	BankName string `json:"bank_name"`
+	Bic      string `json:"bic"`
+	Iban     string `json:"iban"`
+}
```

Comment:
```suggestion
type ChargePaymentMethodDetailsSepaCreditTransfer struct {
	BankName string `json:"bank_name"`
	BIC      string `json:"bic"`
	IBAN     string `json:"iban"`
}
```

Reply:
Actually nevermind, we don't capitalize these acronyms anywhere and should be consistent with elsewhere.

Reply:
Actually nevermind, we don't capitalize these acronyms anywhere and should be consistent with elsewhere.